rpc/errors.go

```diff
@@ -30,6 +32,10 @@ var (
 		code:    1,
 		message: "Failed to write transaction",
 	}
+	ErrNoTraceAvailable = &RPCError{
```

Comment:
The data field needs added to the errors: https://github.com/starkware-libs/starknet-specs/blob/v0.3.0/api/starknet_trace_api_openrpc.json#L330